modules/dxp/apps/search-experiences/search-experiences-web/src/main/resources/META-INF/resources/sxp_blueprint_admin/js/edit_sxp_blueprint/EditSXPBlueprintForm.js

```diff
@@ -477,8 +508,8 @@ function EditSXPBlueprintForm({
 			sortConfiguration: sortConfig ? JSON.parse(sortConfig) : {},
 		};
 
-		if (indexConfig) {
```

Comment:
The acceptance criteria says that if company index was being used, it'd default to empty. Was wondering if that meant defining `indexConfiguration` at all? Also would the default value be `{indexName: ''}`? Though I'm not sure if it'd ever reach that part.